### ISSUE TITLE
Provide endpoint to delete registered query in data audit

### ISSUE THREAD

**Issue by martinweiler** [apache/incubator-kie#7007]
Following up on apache/incubator-kie-issues#834 an endpoint should be provided that allows to delete a query that has been registered previously.

**Comment by elguardian** [comment]
duplicated.
https://github.com/apache/incubator-kie/issues/6969

**Comment by martinweiler** [comment]
I don't think this is a duplication of apache/incubator-kie#6969. The use case for the other issue is "delete audit data older than X" whereas this issue is about providing the user with the ability to delete a specific query (eg. one that has been registered by mistake or one that is no longer needed).
